src/login7-payload.ts

```diff
@@ -412,16 +417,14 @@ class Login7Payload {
       }
     }
 
-    if (this.tdsVersion >= versions['7_4']) {
```

Comment:
This condition is moved outside of the buildFeatureExt method because the entire thing should only be called for TDS >= 7.4.